### DISCUSSION TITLE
Pools vs Prefixes

### DISCUSSION THREAD

**Discussion by jscmenezes** [netbox-community/netbox#5423] [Q&A] [answered]
There is a way to create a Pool independently of the prefix? For example, a pool from 192.168.1.12 to 192.168.1.15 inside a prefix 192.168.1.0/24?
Pools must be limited with a CIDR?

**Comment by jeremystretch** [comment, marked as answer]
NetBox does not currently support modeling any subset of IP addresses within a prefix, but it has been proposed in #834.
